src/Kestrel.Core/KestrelServerLimits.cs

```diff
@@ -251,6 +252,25 @@ public TimeSpan RequestHeadersTimeout
             }
         }
 
+        /// <summary>
+        /// Limits the number of concurrent request streams per HTTP/2 connection. Excess streams will be refused.
+        /// <para>
+        /// Defaults to 100
+        /// </para>
+        /// </summary>
+        public int MaxStreamsPerConnection
```

Comment:
What do you think about making an Http2Limits class/property and adding this to that?

Reply:
No, there won't be that many. Also most of the existing limits apply to Http/2.

Reply:
But for Http/2 specific limits, maybe we should prefix them to something like `HTTP2MaxStreamsPerConnection`?

Reply:
What all do we expect? I can already think of these potentially:

- MaxStreamsPerConnection
- TotalMaxStreams
- ConnectionWindowSize
- StreamWindowSize
- WindowUpdateInterval
- MaxFrameSize

Even if we only do a couple of these, I like having non-HTTP/1.x-related limits separated. In the future, we could add even more related to header table size and possibly other things we haven't even considered yet.

Reply:
I asked @shirhatti and he wants to keep it as a flat list.